File: algorithm/selection/sort.go
package main

import (
	"fmt"
)

func Selection(slice []int){

	for i := 1; i < len(slice); i++ {
		min := i-1
		for j := i; j < len(slice); j++ {

			if slice[min] > slice[j] {
				min = j
			}
		}
		slice[i-1],slice[min] = slice[min],slice[i-1]
	}
}

func main()  {
	slice := []int{2,3,1,4,32,42,88,21,6,8,9,5,14,12,18,54,7,35,36,24,13}
	Selection(slice)
	fmt.Println(slice)
}

File: algorithm/selection/sort.py
import random


def selection(list):
    num = len(list)

    for i in range(num - 1):
        min = i
        for j in range(i + 1, num):
            if list[j] < list[min]:
                min = j

        list[i], list[min] = list[min], list[i]


list = []
for i in range(1, 51):
    list.append(i)

random.shuffle(list)
selection(list)
print(list)
